packages/client/src/workspace/utils.ts

```diff
@@ -4,25 +4,39 @@ import { firstValueFrom } from "rxjs"
 import { currencyPairSymbols$ } from "@/services/currencyPairs"
 
 import { getApps, getSnapshots, getViews } from "./apps"
-import { VITE_RT_URL } from "./constants"
+import { VITE_RT_URL, WS_BASE_URL } from "./constants"
 import { ADAPTIVE_LOGO } from "./home/utils"
+import { getCurrentUser, USER_TRADER } from "./user"
 
 export const getAllMainApps = async (): Promise<App[]> => [
   ...getApps(),
   ...getViews(),
   ...getSnapshots(),
 ]
 
+export const getAllApps = async (): Promise<App[]> => [
+  ...getApps(),
+  ...getViews(),
+  ...getSnapshots(),
+  ...(await getSpotTileApps()),
+]
+
 export const getSpotTileApps = async (): Promise<App[]> => {
+  const currentUser = getCurrentUser()
+  if (currentUser !== USER_TRADER) {
+    return []
+  }
   const currencyPairs = await firstValueFrom(currencyPairSymbols$)
 
   return currencyPairs.map((symbol) => ({
     appId: `reactive-trader-${symbol}`,
+    title: `${symbol} Spot Tile`,
+    description: `Reactive Trader - Live rates and trade ${symbol} in this view`,
     manifestType: "url",
```

Comment:
bit of an anomaly, launched as a view but this is not a `manifestType` known to the OpenFin types, but need this to open a view without a manifest (as the ccypairs are dynamic) .. guessing some of this as this work predated my time on RT